src/controller/event/handler/internal/project.go

```diff
@@ -39,16 +40,15 @@ func (a *ProjectEventHandler) IsStateful() bool {
 
 func (a *ProjectEventHandler) onProjectDelete(ctx context.Context, event *event.DeleteProjectEvent) error {
 	log.Infof("delete project id: %d", event.ProjectID)
-	// delete tag immutable
-	err := immutable.Ctr.DeleteImmutableRuleByProject(ctx, event.ProjectID)
-	if err != nil {
+	if err := immutable.Ctr.DeleteImmutableRuleByProject(ctx, event.ProjectID); err != nil {
```

Comment:
keep the comment `// delete tag immutable`, `// delete tag retention` ?

Reply:
The meaning already exists in the method name.